### ISSUE TITLE
Changes in the layout of blocks don't end up in project config file.

### ISSUE THREAD

**Issue by jan-dh** [verbb/vizy#26]
**Description**
Saving a layout does not end up in config yaml file.

**Steps to reproduce**
1. Created a Vizy field
2. Make a group
3. Add a couple of blocks
4. Add field to the layout of the blocks
5. Save fields 
6. Reopen the Vizy field and add new fields to the layout
7. Save field
8. No changes appear in the project config yam files.

It seems that changes in the layout of a block aren't being picked up

**Additional info**
- Plugin version: 1.0.1
- Craft version: 3.6.15
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jan-dh** [comment]
Can confirm this is now fixed with the `dev-craft-3` branch

**Comment by engram-design** [comment]
Fixed in [1.0.2](https://github.com/verbb/vizy/releases/tag/1.0.2)
